### PR TITLE
Add DateInput interface specification

### PR DESCRIPTION
**Pull request by BigRoma**

### CODE REVIEW

File: DateInput/README.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Functions
+
+`select` - prints prompt message to the user and returns dates entered by user.
+
+arguments:
+
+	answerId: uint32
+    prompt: bytes
+    min: uint32
```
Comment (sertony) [inline DateInput/README.md:23]: Date can a have a negative timestamp

File: DateInput/README.md [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    max: uint32
+    exclude: uint32[]
```
Comment (sertony) [inline DateInput/README.md:25]: If you exclude the date, you exclude the range or an array of ranges, not an array of values.

File: DateInput/README.md [hunk 3]
```diff
@@ -0,0 +1,63 @@
...
+
+`range` - prints prompt message to the user and returns date range entered by user.
+
+arguments:
+
+	answerId: uint32
+    prompt: bytes
+    min: uint32
+    max: uint32
+    exclude: uint32[]
```
Comment (sertony) [inline DateInput/README.md:41]: Actually if you have an exclude here you cannot guaranty the solid range...

I'd consider removing the `exclude` argument for now in both methods, but we might need to discuss together.

**Review by sertony — CHANGES_REQUESTED** [review]

**Review by Keshoid — COMMENTED** [review]
My suggestion:
rename iface to `DateTimeInput`.
remove `exclude` - it complicates the interface and has some corners.
return not just a `uint32` value but a struct with info about time:
```
struct DateTime {
 uint16 year;
uint8 month;
uint8 day;
uint8 hours;
uint8 minutes;
uint8 seconds;
int32 unixtime;
}

```

**Comment by BigRoma** [comment]
> My suggestion:
> rename iface to `DateTimeInput`.
> remove `exclude` - it complicates the interface and has some corners.
> return not just a `uint32` value but a struct with info about time:
> 
> ```
> struct DateTime {
>  uint16 year;
> uint8 month;
> uint8 day;
> uint8 hours;
> uint8 minutes;
> uint8 seconds;
> int32 unixtime;
> }
> ```

Ok, but I don't understand why we must add unix time, it doesn't match with answer structure, because we can set only date, but don't set time. I think we need three methods: `date`, `dateTime`, `time`, that return  different answer types.

**Comment by NoamDev** [comment]
> My suggestion:
> rename iface to `DateTimeInput`.
> remove `exclude` - it complicates the interface and has some corners.
> return not just a `uint32` value but a struct with info about time:
> 
> ```
> struct DateTime {
>  uint16 year;
> uint8 month;
> uint8 day;
> uint8 hours;
> uint8 minutes;
> uint8 seconds;
> int32 unixtime;
> }
> ```

years should be a signed integer.

**Pull request closed without merging**